sklearn/preprocessing/data.py

```diff
@@ -1673,7 +1676,7 @@ def add_dummy_feature(X, value=1.0):
         return np.hstack((np.ones((n_samples, 1)) * value, X))
 
 
-def _transform_selected(X, transform, selected="all", copy=True):
+def _transform_selected(X, transform, selected="all", copy=True, order=False):
```

Comment:
In certain functions like `check_array`, `order` is a keyword argument which describes how the array is ordered.

I'd instead rename this parameter to something like `retain_ordering`.